app/uk/gov/hmrc/cdsreimbursementclaimfrontend/models/address/Address.scala

```diff
@@ -54,7 +57,16 @@ object Address {
     line5: Option[String],
     postcode: Option[String],
     country: Country
-  ) extends Address
+  ) extends Address {
+    def allNonEmptyLines: List[String] = List(
```

Comment:
if we are not passing any address info to BARS, please remove all that code that you did around that.

Reply:
I removed the address lines. 
Be aware though that passing in a BarsAdress is not optional though, even if I don't put the address lines in

Reply:
Yeah I saw that :man_shrugging: